lib5k/src/main/java/io/github/frc5024/purepursuit/pathgen/BezierPath.java

```diff
@@ -0,0 +1,214 @@
+package io.github.frc5024.purepursuit.pathgen;
+
+import java.util.ArrayList;
+import ca.retrylife.ewmath.MathUtils;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import edu.wpi.first.wpilibj.util.Units;
+
+/**
+ * This class is used to generate points along a bezier curves
+ */
+public class BezierPath extends Path{
+
+	// Maximum space allowed between points
+	private double maxSeperation = Units.inchesToMeters(4);
```

Comment:
I would make this:

```java
private static final double DEFAULT_WAYPOINT_SPACING = Units.inchesToMeters(4);
```

Reason 1: This should be a constant, so its marked `final`

Reason 2: When we are following a path at top speed (it was common for teams to hit 20 feet/second during auto in 2018 for example), we want a big spacing (like, a foot or two) because there is no point in having that much precision in a fast path